marabunta/html/migration.html

```diff
@@ -0,0 +1,118 @@
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <title>odoo maintenance</title>
+  <meta name="viewport" content="width=device-width">
+  <meta http-equiv="refresh" content="10" />
+  <style>
+    html {
+      font-family: sans-serif;
+      line-height: 1.15;
+      -webkit-text-size-adjust: 100%;
+    }
+    body {
+      color: #fff;
+      background-color: #21232a;
+      text-align: center;
+      text-shadow: 0 2px 4px rgba(0,0,0,.5);
+      -webkit-box-shadow: inset 0 0 75pt rgba(0,0,0,.8);
+      box-shadow: inset 0 0 75pt rgba(0,0,0,.8);
+      font-family: "Open Sans",Arial,sans-serif;
+      margin: 0;
+      padding: 0;
+      min-height: 100%;
+      display: table;
+      width: 100%;
+      height: 100%;
+    }
+    .cover {
+      display: table-cell;
+      vertical-align: middle;
+      padding: 0 20px;
+    }
+    h1 {
+      margin: .67em 0;
+      font-family: inherit;
+      font-weight: 500;
+      line-height: 1.1;
+      color: inherit;
+      font-size: 36px;
+
+    }
+    .message {
+      color: silver;
+      font-size: 21px;
+      line-height: 1.4;
+    }
+
+    /* loader from https://projects.lukehaas.me/css-loaders/ */
+    .loader,
+    .loader:before,
+    .loader:after {
+      background: #a24689;
+      -webkit-animation: load1 1s infinite ease-in-out;
+      animation: load1 1s infinite ease-in-out;
+      width: 1em;
+      height: 4em;
+    }
+    .loader {
+      color: #a24689;
+      text-indent: -9999em;
+      margin: 88px auto;
+      position: relative;
+      font-size: 11px;
+      -webkit-transform: translateZ(0);
+      -ms-transform: translateZ(0);
+      transform: translateZ(0);
+      -webkit-animation-delay: -0.16s;
+      animation-delay: -0.16s;
+    }
+    .loader:before,
+    .loader:after {
+      position: absolute;
+      top: 0;
+      content: '';
+    }
+    .loader:before {
+      left: -1.5em;
+      -webkit-animation-delay: -0.32s;
+      animation-delay: -0.32s;
+    }
+    .loader:after {
+      left: 1.5em;
+    }
+    @-webkit-keyframes load1 {
+      0%,
+      80%,
+      100% {
+        box-shadow: 0 0;
+        height: 4em;
+      }
+      40% {
+        box-shadow: 0 -2em;
+        height: 5em;
+      }
+    }
+    @keyframes load1 {
+      0%,
+      80%,
+      100% {
+        box-shadow: 0 0;
+        height: 4em;
+      }
+      40% {
+        box-shadow: 0 -2em;
+        height: 5em;
+      }
+    }
+  </style>
+</head>
+<body>
+  <div class="cover">
+    <div class="loader">Waiting...</div>
+    <h1>odoo maintenance</h1>
+    <p class="message">An upgrade of the application is currently in progress... come back later.</p>
```

Comment:
*please* come back later ?

Reply:
To me, 'please' has become meaningless now because it's used everywhere in computer messages. Sometimes I almost feel it like an hypocritical message, coming from a computer. I changed it though.